client/state/sites/reducer.js

```diff
@@ -71,9 +74,12 @@ export function items( state = {}, action ) {
 			// Normalize incoming site(s) to array
 			const sites = action.site ? [ action.site ] : action.sites;
 
+			if ( SITES_UPDATE === action.type && state === null ) {
+				return state;
+			}
```

Comment:
why are we aborting here instead of adding the site information for those sites which are being updated? do we not have enough information for that? seems like if our `state === null` but we are receiving updates then we could create the necessary state by "applying" the update to "nothing"